### PR TITLE
Update to FRETBursts 0.7

### PR DESCRIPTION
**Pull request by tritemio**
Try 2

### CODE REVIEW

**Comment by djsutherland** [comment]
Everything currently in `build` should go in `host`. Python packages usually don't have anything in `build` in conda-build 3 except maybe `{{ compiler('c') }}` (which you also need).

**Comment by djsutherland** [comment]
@conda-forge-admin, please rerender

@tritemio I went ahead and made the changes directly since it's probably easier than describing what to do. :)

**Comment by tritemio** [comment]
@dougalsutherland, awesome, thanks a lot! Sorry but I went offline yesterday. The changes make sense, I was not up to date with the new conda-build 3.

The builds now only fail on windows due to failed import of pytables:

```
(C:\bld\fretbursts_1532446216845\_test_env) C:\bld\fretbursts_1532446216845\test_tmp>"C:\bld\fretbursts_1532446216845\_test_env\python.exe" -s "C:\bld\fretbursts_1532446216845\test_tmp\run_test.py" 
import: 'fretbursts'
Traceback (most recent call last):
  File "C:\bld\fretbursts_1532446216845\test_tmp\run_test.py", line 2, in <module>
    import fretbursts
  File "C:\bld\fretbursts_1532446216845\_test_env\lib\site-packages\fretbursts\__init__.py", line 129, in <module>
    from . import loader, hdf5, select_bursts, fretmath
  File "C:\bld\fretbursts_1532446216845\_test_env\lib\site-packages\fretbursts\loader.py", line 22, in <module>
    import tables
  File "C:\bld\fretbursts_1532446216845\_test_env\lib\site-packages\tables\__init__.py", line 93, in <module>
    from .utilsextension import (
ImportError: DLL load failed: The specified module could not be found.
```
 Is pytables on windows currently broken on conda-forge? Is there a known working version I can pin?

**Comment by tritemio** [comment]
Suggestions on solving this pytables issue on windows appreciated @conda-forge/help-python

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/tritemio/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/OpenSMFS/{{ name }}/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   skip: true  # [win32]
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: We don't build win32 any more

Comment (tritemio) [reply recipe/meta.yaml]: @CJ-Wright, thanks I'll remove it.

**Comment by CJ-Wright** [comment]
Is this relevant? https://github.com/conda-forge/pytables-feedstock/issues/31

**Comment by tritemio** [comment]
Yes, @CJ-Wright, this looks the exact same issue here. 

Seems that pytables is currently broken on windows at least since 3.4.3_8 due to not finding blosc DLL at import time.

Using pytables 3.3.* fixed the build. Any other workaround I could try?

**Comment by CJ-Wright** [comment]
No idea, you could follow up with the pytables group.

**Pull request merged**